docs/Taps.md

```diff
@@ -17,7 +17,7 @@ dunn/emacs
 ```
 
 * `brew tap <user/repo>` makes a shallow clone of the repository at
-  https://github.com/user/repo. After that, `brew` will be able to work on
+  https://github.com/user/repository. After that, `brew` will be able to work on
```

Comment:
This should be left as-is

Reply:
I thought so too. The robots can't rule us all the time! I'll work on this later.